capstone/src/main/java/com/google/sps/data/ApkFileFeatures.java

```diff
@@ -25,4 +25,18 @@ public ApkFileFeatures(Entity entity) {
 
   }
 
+  public ApkFileFeatures(List<Long> dexFiles, List<Long> resFiles, List<Long> libFiles, List<Long> assetsFiles, List<Long> resourcesFiles, List<Long> miscFiles, Long numOfFiles, Long totalSize, Long sizeLost) {
```

Comment:
Break this line up for readability.
```
public ApkFileFeatures(List<Long> dexFiles, List<Long> resFiles, List<Long> libFiles, List<Long> assetsFiles, List<Long> resourcesFiles, List<Long> miscFiles, Long numOfFiles, Long totalSize, Long sizeLost) {
```

to

```
public ApkFileFeatures(List<Long> dexFiles,
                       List<Long> resFiles,
                       List<Long> libFiles,
                       List<Long> assetsFiles,
                       List<Long> resourcesFiles,
                       List<Long> miscFiles,
                       Long numOfFiles,
                       Long totalSize,
                       Long sizeLost) {
```

Reply:
Thank you!